### PR TITLE
[Method References 2] LPS-165626

### PR DESCRIPTION
**Pull request by JanisZhang**
previous: https://github.com/liferay-core-infra/liferay-portal/pull/2528 . Thanks!

### CODE REVIEW

File: modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/transformer/test/JournalTransformerTest.java [hunk 1]
```diff
@@ -409,6 +404,9 @@ private void _testTransformDDMFormField(
 	@Inject
 	private Portal _portal;
 
+	@Inject(filter = "component.name=*.RegexTransformerListener")
```
Comment (dantewang) [inline modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/transformer/test/JournalTransformerTest.java:407]: @JanisZhang Could you use the full class name here?

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

File: modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/transformer/test/JournalTransformerTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/transformer/test/JournalTransformerTest.java]: Hi @dantewang, force-pushed. thx

**Review by dantewang — APPROVED** [review]

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
@JanisZhang Please remove JournalPropertiesTransformerListener and resend

**Pull request closed without merging**